cmd/opampsupervisor/supervisor/supervisor.go

```diff
@@ -309,24 +310,27 @@ func initTelemetrySettings(logger *zap.Logger, cfg config.Telemetry) (telemetryS
 }
 
 func (s *Supervisor) Start() error {
```

Comment:
Just to play devil's advocate, do we always want a span for Supervisor startup? Most of these issues should be uncommon, and the call stack is fairly shallow (`main -> (*Supervisor).Start`), so I think we either end up with an empty span, or one that more or less directly mirrors the log the Supervisor will print.

I don't have enough expertise with the Supervisor in production environments to really make a call as to whether this will be useful, so if anyone else has an opinion that would be welcome, but I want to make sure the telemetry we're emitting is useful.